docs/password.rst

```diff
@@ -0,0 +1,56 @@
+.. Information about how to securely store passwords
+
+.. _password:
+
+=================================
+Securing your password
+=================================
+
+Your Data Delivery System (DDS) password will be used to protect your data from any unauthorized parties. Choosing a secure password is therefore essential since the complexity will determine how easy it is to guess the password and thereby gain access to your data. Remembering a number of different passwords to different services can be difficult, and many people resort to using the same or similar password for all, or saving them in insecure ways where they are easily lost or/and read by others.
```

Comment:
```suggestion
Your Data Delivery System (DDS) password will be used to protect your data from any unauthorized parties. Choosing a secure password is therefore essential since the password's complexity will determine how easy it is to guess the password and thereby gain access to your data. Remembering a number of different passwords for different services can be difficult, and many people resort to using the same or similar password for all, or saving them in insecure ways where they are easily lost or/and read by others.
```